File: crates/tldr-cli/tests/fixtures/vuln_migration_v1/php/command_injection_string_literal_fp.php
<?php
// vt=CommandInjection lang=php — names below are inside strings/comments only
// $_GET is a source. mysqli_query, echo, shell_exec, fopen,
// file_get_contents, unserialize are sinks. Referenced in strings only below.

function docs() {
    $doc = '$_GET[id] flows into mysqli_query(SELECT ... )';
    $more = 'shell_exec, fopen, file_get_contents, unserialize, echo — string-only';
    return $doc . $more;
}

File: crates/tldr-cli/tests/fixtures/vuln_migration_v1/swift/command_injection_string_literal_fp.swift
// vt=CommandInjection lang=swift — names below are inside strings/comments only
// request.query[...] is a source. executeQuery, Process.launchedProcess,
// FileHandle are sinks — referenced in strings only here.

import Foundation

func docs() -> String {
    let doc = "request.query[id] flows into stmt.executeQuery(SELECT ... )"
    let more = "Process.launchedProcess, FileHandle(forReadingAtPath:) — string-only"
    return doc + more
}
